cmd/commands/common.go

```diff
@@ -282,23 +288,37 @@ func ensureGitToken(cmd *cobra.Command, cloneOpts *git.CloneOptions, verify bool
 
 func ensureGitPAT(cmd *cobra.Command, opts *RuntimeInstallOptions) error {
 	var err error
-	if !store.Get().Silent {
-		return getGitPATFromUserInput(cmd, opts)
+	tokenFromFlag, err := cmd.Flags().GetString("personal-git-token")
```

Comment:
This should be taken from the `opts *RuntimeInstallOptions` like the rest of the `ensure..()` functions. If tomorrow we change the flag's name it would be a pain.